src/mixins/metadata.js

```diff
@@ -9,11 +9,20 @@
 */
 
 import { addOrUpdateMetadata } from 'docc-render/utils/metadata';
+import { firstParagraph } from 'docc-render/utils/strings';
 import ContentNode from 'docc-render/components/ContentNode.vue';
 
 export default {
   methods: {
-    extractText: ContentNode.methods.extractText,
+    // Extracts the first paragraph of plaintext from the given content tree,
+    // which can be used for metadata purposes.
+    extractText(content = []) {
```

Comment:
should we call it `extractFirstParagraph`? Probably its not a big deal if we leave it...

Reply:
cf23e9e2e9770ff8360f102cd4f86154e183c1ac